### ISSUE TITLE
Presenting PopUpBar changes TabBar color since 2.12.1

### ISSUE THREAD

**Issue by MacMark** [LeoNatan/LNPopupController#514]
**Describe the Bug**
When using presentPopupBarWithContentViewController the TabBar's color is changed since version 2.12.1. Before that version the tabbar keeps its color.

**To Reproduce**
1. Use the attached demo project
[PopupMinTest.zip](https://github.com/LeoNatan/LNPopupController/files/9471109/PopupMinTest.zip)
2. Click on a tabbar item to show the popup
3. See the color of the tab changing
4. Click on a tabbar item again to hide the popup
5. See the original tabbar color returning
6. Click again to repeat the problem
7. Change the LNPopUpController framework's version to 2.12.0 like shown in other screenshot
8. Repeat above steps and see the tabbar keeps its color when the popup appears

**Expected Behavior**
Tabbar keeps color when popupbar appears like it did with 2.12.0 and earlier.

**Screenshots**
<img width="645" alt="lnpopup-version-settings" src="https://user-images.githubusercontent.com/626963/187948686-ccd08e97-5ca3-4cbf-b15f-9191c6996778.png">

https://user-images.githubusercontent.com/626963/187948702-ef9fb840-8daa-46d7-aadf-751c865a52c5.mp4

https://user-images.githubusercontent.com/626963/187950482-0c92f833-d679-4f03-80e4-6373f3ca4d67.mp4

**Additional Context**
Nothing yet.

**Comment by LeoNatan** [comment]
How are you setting the tab bar’s color? You should be using the modern [`UIBarAppearance`](https://developer.apple.com/documentation/uikit/uibarappearance?language=objc) API, which is the only customization supported by `LNPopupController`.

**Comment by LeoNatan** [comment]
Also, what iOS version are we talking about?

**Comment by MacMark** [comment]
I did not try UIBarAppearance yet because the app supports iOS 12. Thanks for the hint, i will try and add this (for 13+ devices) and then report the outcome here.
Currently i just use `self.view.backgroundColor = [UIColor darkGrayColor];` to set the tab bar's color.

The tested versions are the iOS 16 and Xcode 14 betas.

**Comment by LeoNatan** [comment]
Note that LNPopupController only supports iOS 13 and above since a while ago.

**Comment by LeoNatan** [comment]
https://github.com/LeoNatan/LNPopupController/issues/456

**Comment by MacMark** [comment]
I added usage of UITabBarAppearance and this seems to keep the color / fix the problem according to a first test:
    ```
self.view.backgroundColor = [UIColor grey100FixColor];
    if (@available(iOS 13.0, *)) {
        UITabBarAppearance *appearance = [[UITabBarAppearance alloc] init];
        appearance.backgroundColor = self.view.backgroundColor;
        self.tabBar.standardAppearance = appearance;
    } else {
        // Fallback on earlier versions
    }
```
Thank you for your help and support, Leo!

**Comment by LeoNatan** [comment]
C
